### ISSUE TITLE
Strong name signature not valid for this assembly FFmpeg.AutoGen.Abstractions.dll

### ISSUE THREAD

**Issue by timotech** [AddictedCS/soundfingerprinting#211]
Hi everyone,
Has anybody faced this issue during deployment with the error below:
Strong name signature not valid for this assembly FFmpeg.AutoGen.Abstractions.dll

This happens when I do click once deployment, and the application doesn't get installed.
The issue occurs when I'm using ffmpeg for fingerprinting. Everything works fine, but gives error above during deployment.

**Comment by timotech** [comment]
Apparently, I didn't need it for my app. Please ignore the usage

**Comment by AddictedCS** [comment]
This error is not related to `SoundFingerprinting`. I'm not familiar with your setup and can't help.
Try clearing global assembly cache, as this error might generally happen during a version mismatch between installed dependencies and those in the GAC.

**Comment by timotech** [comment]
Hi Sergiu,
Wow I've been trying to reach you. You are quite hard to reach. Thanks for your comment. I know its not a `SoundFingerprinting` issue. But I noticed that when you download `Soundfingerprinting.emy` through nuget, it comes with that dll, `FFmpeg.AutoGen.Abstractions`. And anytime I tried to do clickonce and install the program on my system, then that error pops up. I wanted to know if anyone has faced such issue. I will try your suggestion and give feedback. Thanks
